tests/posix/fs/prj.conf

```diff
@@ -8,3 +8,4 @@ CONFIG_POSIX_FS=y
 CONFIG_ZTEST=y
 CONFIG_ZTEST_NEW_API=y
 CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_EVENTFD=n
```

Comment:
Is the main reason for n-selecting this to just not link in unused code? Just curious.

Reply:
That, and the test case failed on one of the arc qemu platforms w/o this.  So guessing there was a code size growth or something impact that platform.